rebar.lock

```diff
@@ -2,16 +2,16 @@
 [{<<"base64url">>,{pkg,<<"base64url">>,<<"1.0.1">>},0},
  {<<"cowlib">>,{pkg,<<"cowlib">>,<<"2.7.3">>},1},
  {<<"gun">>,{pkg,<<"gun">>,<<"1.3.3">>},0},
```

Comment:
@sedinin ,you've bump gun from 1.3.3 to 2.1.0 but you didn't update the rebar.lock.

Reply:
yeah, my bad, will fix